apps/routes/lib/route.ex

```diff
@@ -8,6 +8,7 @@ defmodule Routes.Route do
             name: "",
             long_name: "",
             color: "",
+            sort_order: :unknown,
```

Comment:
It looks like this isn't currently an allowed value according to the typespec. But does the API always return a `sort_order ` value for every route? Maybe we can just not specify a default value here?

Reply:
I'll supply a better default value...